src/localization/strings.ts

```diff
@@ -0,0 +1,24 @@
+import LocalizedStrings, { type LocalizedStringsMethods } from "localized-strings";
+import en from "./en.json";
+import fi from "./fi.json";
+
+/**
+ * Type representing the shape of your translation strings
+ */
+interface TranslationStrings {
+  notYetImplemented: string
+}
+
+/**
+ * Interface describing localized strings
+ */
+export interface IStrings extends LocalizedStringsMethods, TranslationStrings {
```

Comment:
Again this seems a little different to the example in OSS which should be the guide to follow.
e.g.export interface IStrings extends LocalizedStringsMethods, TranslationStrings could be 
export interface Localized extends LocalizedStringsMethods

const strings: IStrings = new LocalizedStrings could be
const strings: Localized = new LocalizedStrings

So really just a difference in naming, but I'd recommend keeping the names the same as they are in other projects,